datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Ownership/EditOwnersModal.tsx

```diff
@@ -78,10 +78,25 @@ export const EditOwnersModal = ({
     const renderSearchResult = (entity: Entity) => {
         const avatarUrl =
             (entity.type === EntityType.CorpUser && (entity as CorpUser).editableProperties?.pictureLink) || undefined;
+        const isCorpUser = (entity.type as EntityType) === EntityType.CorpUser;
+        const displayCorpUserDepartmentName =
+            (entity.type === EntityType.CorpUser && (entity as CorpUser).properties?.departmentName) || '';
+        const displayCorpUserUserName = (entity.type === EntityType.CorpUser && (entity as CorpUser).username) || '';
+    
         const displayName = entityRegistry.getDisplayName(entity.type, entity);
+        const concatenatedValue = isCorpUser
```

Comment:
Question: What about the user's _title_ (e.g. Software Engineer, Data Analyst, etc).

This is ALSO something we might have available to us, so perhaps we can use that after or before the department?

Reply:
Addressed